From 2a1ee7ba1254b10638e1973a169992c4bb5ff37b Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Thu, 19 Aug 2010 11:24:45 -0400
Subject: [PATCH] perf: force include of <stdbool.h>

Large parts of perf use bool variables and assignments.
Rather than depending on transient includes, we force it
by including <stdbool.h> directly in perf.h

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 tools/perf/perf.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index e004178472d9e..00ae0f27bd1e2 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -1,5 +1,7 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 #ifndef _PERF_PERF_H
+#include <stdbool.h>
+
 #define _PERF_PERF_H
 
 #define MAX_NR_CPUS			4096
-- 
2.39.2

